.circleci/config.yml

```diff
@@ -1,557 +0,0 @@
-#
-# Defaults
-#
-
-# shared
-
-defaults: &defaults
-  working_directory: ~/italia-app
-
-node_cache_key: &node_cache_key v1.17-dependencies-{{ checksum "yarn.lock" }}
-
-android_sdk_cache_key: &android_sdk_cache_key v1-android-sdkmanager-packages-{{ checksum "scripts/circleci-android-setup.sh" }}
-
-android_jars_cache_key: &android_jars_cache_key jars-{{ checksum "android/build.gradle" }}-{{ checksum  "android/app/build.gradle" }}
-
-ios_gems_cache_key: &ios_gems_cache_key ruby-{{ checksum "Gemfile.lock" }}
-
-release_tag: &release_tag /^[0-9]+\.[0-9]+\.[0-9]+(-rc\.[0-9]+)?$/
-
-# nodejs builds
-defaults_js: &defaults_js
-  <<: *defaults
-  docker:
-    - image: cimg/node:16.19.0
-
-# android builds
-defaults_android: &defaults_android
-  <<: *defaults
-  docker:
-    - image: cimg/android:2022.09.2-node
-  environment:
-    TERM: "dumb"
-    ANDROID_SDK_BUILD_TOOLS_REVISION: "29.0.3"
-    ANDROID_SDK_BUILD_API_LEVEL: "30"
-    ANDROID_GOOGLE_API_LEVEL: "23"
-    # - REACT_NATIVE_MAX_WORKERS: 1
-    GRADLE_OPTS: '-Dorg.gradle.daemon=false -Dorg.gradle.jvmargs="-Xmx8g -XX:MaxPermSize=4096m -XX:+HeapDumpOnOutOfMemoryError"'
-
-# macos builds
-defaults_macos: &defaults_macos
-  <<: *defaults
-  macos:
-    xcode: "14.2.0"
-  resource_class: macos.x86.medium.gen2
-  working_directory: /Users/distiller/italia-app
-  environment:
-    # Fastlane requires locale set to UTF-8
-    # see https://docs.fastlane.tools/getting-started/ios/setup/#set-up-environment-variables
-    LC_ALL: en_US.UTF-8
-    LANG: en_US.UTF-8
```

Comment:
Is this the default in GA or should we set it explicitly?

Reply:
nice catch I'll add them anyway as static env vars

Reply:
see [2e1fef6](https://github.com/pagopa/io-app/pull/4609/commits/2e1fef68999aede8a458c73812622842ee9ec510)